staged_dependencies.yaml

```diff
@@ -27,12 +24,6 @@ downstream_repos:
   insightsengineering/teal.modules.general:
```

Comment:
Downward dependency
As of this https://github.com/insightsengineering/teal.transform/issues/133 scda is going to be decoupled from teal.trasform once merged . this should be removed